src/test/models/user.ts

```diff
@@ -1,6 +1,7 @@
 import * as mongoose from 'mongoose';
 import * as _ from 'lodash';
-import * as findOrCreate from 'mongoose-findorcreate';
+// tslint:disable-next-line:no-var-requires
+const findOrCreate = require('mongoose-findorcreate');
```

Comment:
Yes, this package has no typings, so you can't import non TS module with this syntax. It won't compile when `target` is ES6.
https://www.npmjs.com/search?q=mongoose-findorcreate

And the target has to be `ES6` because you are using ES6 feature called `Function.name` 😉